injective/assetlist.json

```diff
@@ -219,6 +219,47 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/WOSMO.png"
       }
+    },
+    {
+      "description": "GLTO-ERC20 on injective",
+      "denom_units": [
+        {
+          "denom": "peggy0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2",
+          "exponent": 0
+        },
+        {
+          "denom": "glto",
+          "exponent": 6
+        }
+      ],
+      "base": "glto",
```

Comment:
"base" should now also point to "peggy0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2"